### ISSUE TITLE
Show description in explorer view

### ISSUE THREAD

**Issue by gmkado** [microsoft/codetour#292]
Thanks for the awesome extension!

Maybe I'm missing some configuration, but in the explorer sidebar my view looks like:

```
#1 - path/to/first/step.cs
#2 - path/to/second/step.cs
```
It would be great to see the description associated with each step here so we can skip to a specific point in the tour

**Comment by nuorder-vrobison** [comment]
Yes.  Or a Title! I mostly navigate from the explorer, for very complex code paths.  Having a title would be a huge time-saver.

**Comment by hitzhangjie** [comment]
Yes, maybe we could write the discussion like git commit message:

```
heading 
[blankline]
body
```

When displaying in the codetour explorer, it could extract the heading and show it.
